applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -262,6 +263,7 @@ def AssembleTestSuites():
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestPatchTestSmallStrainBbar]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestPatchTestLargeStrain]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestQuadraticElements]))
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestPatchTestSmallDisplacementMixedVolumetricStrain]))
```

Comment:
fast enough for the smallsuite?

Reply:
Yes. This one is fast. Its a patch of 4 (2D) and 8 (3D) elements.